src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/transport/Codecs.kt

```diff
@@ -416,15 +417,23 @@ object GroupBatchCodec : AbstractCodec<GroupBatch>(50u) {
 
 object GroupListCodec : ListCodec<MessageGroup>(51u, MessageGroupCodec)
 
-inline fun ByteBuf.forEachValue(context: DecodeContext, action: (codec: ValueCodec<*>) -> Unit) {
+private val LOGGER = KotlinLogging.logger(UnknownValueCodec::class.java.canonicalName)
```

Comment:
This logger is related to special codec class but name doesn't reflect this aspect. Maybe replace this statement to getter like this
 ```suggestion
private val LOGGER: KLogger
  get() = KotlinLogging.logger(this::class.java.canonicalName)
```
or special method received class as paramether because I'm not sure that this is set for the static getter function

Reply:
There is no correct `this` context in this method so it is impossible to do what you suggest. The only way is to pass the logger into this method (each codec will pass its logger) or pass a codec name into the method. I would prefer the latter one